src/GAP/gap_to_oscar.jl

```diff
@@ -168,7 +168,7 @@ function matrices_over_cyclotomic_field(F::AnticNumberField, gapmats::GapObj)
     Nemo.is_cyclo_type(F) || throw(ArgumentError("F is not a cyclotomic field"))
     GAPWrap.IsList(gapmats) || throw(ArgumentError("gapmats is not a GAP list"))
     GAPWrap.IsEmpty(gapmats) && throw(ArgumentError("gapmats is empty"))
-    GAP.Globals.IsCyclotomicCollCollColl(gapmats) ||
+    GAPWrap.IsCyclotomicCollCollColl(gapmats) ||
```

Comment:
@ThomasBreuer it seems we have no CI test for this conversion case? Can you add one?